crates/core/src/subscription/execution_unit.rs

```diff
@@ -265,10 +267,17 @@ impl ExecutionUnit {
         tx: &'a TxMode<'a>,
         tables: impl 'a + Clone + Iterator<Item = &'a DatabaseTableUpdate>,
     ) -> Result<Option<DatabaseTableUpdateCow<'a>>, DBError> {
+        let slow_query = db
+            .config
+            .read()
+            .slow_query
+            .for_incremental_updates(ctx.sql_or_reducer());
```

Comment:
We probably only want to read the environment variable once before evaluating the queries, so that we don't contend on a mutex here.

Reply:
Done